src/CircuitNotation.hs

```diff
@@ -1213,28 +1149,33 @@ ppr a = do
 showC :: Data.Data a => a -> String
 showC a = show (typeOf a) <> " " <> show (Data.toConstr a)
 
--- ppp :: MonadIO m => String -> m ()
--- ppp s = case SP.parseValue s of
---   Just a -> valToStr a
-
 -- Names ---------------------------------------------------------------
 
+fwdNames :: [String]
+fwdNames = ["Fwd", "Signal"]
```

Comment:
I'd expect writing `Signal` to enforce the type to be Signal. I'm fine with it how it is if that's too much hassle.

Reply:
Yeah it'd be pretty annoying because I would need an inference_helper to do it..